src/components/BundleRow.js

```diff
@@ -26,10 +26,16 @@ import {
 
 const MAX_MESSAGE_COUNT = 25;
 
+// TODO: Using colors sholuld default to false?
```

Comment:
Related to above comment: if using colors should be opt-out, delete this comment. If opt-in, change the next line to set to false.